packages/rollup-plugin-workbox/src/index.ts

```diff
@@ -1,15 +1,23 @@
 import {
   generateSW as _generateSw,
   injectManifest as _injectManifest,
-  GenerateSWConfig,
-  InjectManifestConfig,
+  GenerateSWOptions,
```

Comment:
We can use the `import type {}`?

Reply:
Sure. I will update the PR once we agree about how to proceed regarding the above comment.

Reply:
Done.